src/Microsoft.Sbom.Api/Manifest/Configuration/SbomConfigProvider.cs

```diff
@@ -73,11 +79,6 @@ public class SbomConfigProvider : ISbomConfigProvider
             ILogger logger,
             IRecorder recorder)
         {
-            if (manifestConfigHandlers is null)
```

Comment:
Why is this removed?

Reply:
It is a duplicate null check, there's null check for this right after this statement
```csharp
this.manifestConfigHandlers = manifestConfigHandlers ?? throw new ArgumentNullException(nameof(manifestConfigHandlers));
```